packages/swingset-runner/demo/resolveChain/bootstrap.js

```diff
@@ -9,7 +9,7 @@ export function buildRootObject(_vatPowers, options) {
   function waitFor(who, p) {
     p.then(
       answer => {
-        if (0 < count && count < 50) {
+        if (count > 0 && count < 50) {
```

Comment:
This bug fix is completely orthogonal to this PR, yes? Was it a known bug?

Reply:
This is a style change to appease `eslint`, which insists that the literal be on the right hand side so it doesn’t “sound like Yoda”. It’s debatable. I personally prefer the previous style since the expressions appear in ascending order, but this was the only inconsistency with that rule. I’ll broach the topic.